### ISSUE TITLE
Help, the fullscreen button/function not working

### ISSUE THREAD

**Issue by hamkah2c** [ewasion/userscripts#2]
I just installed this userscript, and the button for fullscreen and fit to width not there anymore.
Screenshot:
![image](https://user-images.githubusercontent.com/8862875/36339017-7311cd3a-13ee-11e8-87cb-2e1e7573dd5d.png)
This is error from the console in google chrome:

ERROR: Execution of script 'MangaDex Reader fullscreen' failed! Cannot read property 'querySelector' of undefined
(anonymous) @ chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:2
14:09:20.593 chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:2 TypeError: Cannot read property 'querySelector' of undefined
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:181)
    at Window.eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:276)
    at <anonymous>:2:120
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:2)
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:3)
    at Object.eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:277)
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=2f06805b-e9a2-4402-bdb5-5e0b0557cb7a:279)
    at eval (<anonymous>)
    at <anonymous>:2:120
    at Object.b [as F_c] (<anonymous>:1:399)

**Comment by Teasday** [comment]
Yeah, sorry, I haven't bothered updating that script because I've been trying to get Holo, the site's owner, to update the actual reader to incorporate it and some other features. Unfortunately it's been taking quite a while.
